src/enclave/enclave.rs

```diff
@@ -86,11 +81,69 @@ impl Enclave {
     }
 }
 
+/// The registers that can be passed to/from the enclave
+#[repr(C)]
+#[derive(Default, Debug)]
+#[allow(missing_docs)]
+pub struct Registers {
```

Comment:
But it is not missing docs :monocle_face:

Reply:
The fields are. But documenting registers is silly.

Reply:
Oh indeed! I have Friday-brain.